pkg/components/standalone_loader.go

```diff
@@ -54,7 +53,11 @@ func (s *StandaloneComponents) LoadComponents() ([]components_v1alpha1.Component
 	list := []components_v1alpha1.Component{}
 
 	for _, file := range files {
-		if !file.IsDir() && s.isYaml(file.Name()) {
+		if !file.IsDir() {
+			if !s.isYaml(file.Name()) {
```

Comment:
would it better to directly call `utils.IsYaml()` and remove local `isYaml`?

Reply:
Ok, I will change it later.